hooks/useTransactionExport.ts

```
@@ -0,0 +1,285 @@
+import { useState } from 'react';
+import { useDatabase } from '@nozbe/watermelondb/hooks';
+import { TransactionModel } from '../database/transaction-model';
+import { Q } from '@nozbe/watermelondb';
+import Papa from 'papaparse';
+import dayjs from 'dayjs';
+import * as FileSystem from 'expo-file-system';
+import * as Sharing from 'expo-sharing';
+import * as DocumentPicker from 'expo-document-picker';
```

ESLint

[error] 'DocumentPicker' is defined but never used.

(@typescript-eslint/no-unused-vars)